cfme/common/__init__.py

```diff
@@ -355,26 +355,18 @@ def process_field(values):
             return map(SummaryValue, values)
 
 
-class Validatable(SummaryMixin):
-    """
-    Class which Middleware provider and other middleware pages must extend
-    to be able to validate properties values shown in summary page.
-    """
-
-    """
-    Tuples which first value is the provider class's attribute name,
-    the second value is provider's UI summary page field key.
-
-    Should have values in child classes.
+class Validatable(SummaryMixin, Taggable):
```

Comment:
@mfalesni We could not extend `Taggable` here, because `BaseProvider` already extended with `Taggable`, this will result
```
TypeError: Error when calling the metaclass bases
    Cannot create a consistent method resolution
order (MRO) for bases SummaryMixin, Taggable
``` 

We should remove `Taggable` from `BaseProvider` or from here.